Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿using AirportTicketBookingSystem.BookingHandler;
+using AirportTicketBookingSystem.FlightBoard;
+using AirportTicketBookingSystem.FlightHandler;
+using AirportTicketBookingSystem.ManagerHandler;
+using AirportTicketBookingSystem.PassengerHandler;
+using AirportTicketBookingSystem.Validation;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Channels;
+
+namespace AirportTicketBookingSystem
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            AirlineFlightBoard flightBoard = new AirlineFlightBoard();
+
+            while (true)
+            {
+                LogInConsoleMessage();
+                ChooseManagerOrPassenger ManagerOrPassenger = GetManagerOrPassenger();
```

Comment:
we can do that by having some dummy users each with a specific type, then once a user log in we decide which console to display based on the user type.